### PR TITLE
Generate a few ext/standard methods synopses from stubs - part 2

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

File: reference/stream/functions/stream-socket-get-name.xml [hunk 1]
```diff
@@ -46,7 +46,7 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   The name of the socket.
+   The name of the socket.,&return.falseforfailure;.
```
[the remark this answers is not in the document]
Comment (kocsismate) [reply reference/stream/functions/stream-socket-get-name.xml]: Thanks! Fixed!

File: reference/url/functions/get-headers.xml [hunk 2]
```diff
@@ -12,7 +12,7 @@
    <type class="union"><type>array</type><type>false</type></type><methodname>get_headers</methodname>
    <methodparam><type>string</type><parameter>url</parameter></methodparam>
    <methodparam choice="opt"><type>bool</type><parameter>associative</parameter><initializer>&false;</initializer></methodparam>
-   <methodparam choice="opt"><type class="union"><type>resource</type><type>null</type></type><parameter>context</parameter><initializer>&null;</initializer></methodparam>
+   <methodparam choice="opt"><type>resource</type><parameter>context</parameter><initializer>&null;</initializer></methodparam>
```
Comment (cmb69) [inline reference/url/functions/get-headers.xml:15]: See https://github.com/php/php-src/pull/7813.

File: reference/url/functions/parse-url.xml [hunk 3]
```diff
@@ -9,7 +9,7 @@
  <refsect1 role="description">
   &reftitle.description;
   <methodsynopsis>
-   <type>mixed</type><methodname>parse_url</methodname>
+   <type class="union"><type>int</type><type>string</type><type>array</type><type>null</type><type>false</type></type><methodname>parse_url</methodname>
```
Comment (cmb69) [inline reference/url/functions/parse-url.xml:12]: déjà vu – or did we merge this very change earlier today. :)

**Review by cmb69 — APPROVED** [review]
Thank you! This looks good except for the `get_header()` change.

File: reference/url/functions/parse-url.xml (hunk 3 above)
Comment (kocsismate) [reply reference/url/functions/parse-url.xml:12]: Hmm, seems like :)

**Pull request merged**